packages/serverpod_shared/test/password_manager_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:serverpod_shared/serverpod_shared.dart';
+import 'package:test/test.dart';
+import 'package:yaml/yaml.dart';
+
+const _defaultPasswordConfig = '''
+shared:
+  mySharedPassword: 'my password'
+
+development:
+  database: 'development db pass'
+  redis: 'development redis pass'
+
+  serviceSecret: 'development service secret'
+
+staging:
+  database: 'staging db pass'
+  serviceSecret: 'staging service secret'
+
+production:
+  database: 'production db pass'
+  serviceSecret: 'production service secret'
+''';
+
+void main() {
+  group(
```

Comment:
Suggestion: 
 * Add a test with an empty passwords file
 * Add a test for a runMode that doesn't exist.